backend/app/api/admin_routes/evaluation/evaluation_dataset.py

```diff
@@ -0,0 +1,233 @@
+
+    session.merge(evaluation_dataset)
+    session.commit()
+
+    return evaluation_dataset
+
+
+@router.get("/admin/evaluation/dataset")
+def list_evaluation_dataset(
+    session: SessionDep,
+    user: CurrentSuperuserDep,
+    params: Params = Depends(),
+) -> Page[EvaluationDataset]:
+    stmt = (
+        select(EvaluationDataset)
+        .where(EvaluationDataset.user_id == user.id)
+        .order_by(desc(EvaluationDataset.id))
+    )
+    return paginate(session, stmt, params)
+
+
+@router.post("/admin/evaluation/dataset-item")
+def create_evaluation_dataset_item(
+    evaluation_dataset_item: ModifyEvaluationDatasetItem,
+    session: SessionDep,
+    user: CurrentSuperuserDep
+) -> EvaluationDatasetItem:
+    evaluation_task_item = EvaluationDatasetItem(
+        query=evaluation_dataset_item.query,
+        reference=evaluation_dataset_item.reference,
+        retrieved_contexts=evaluation_dataset_item.retrieved_contexts,
+        extra=evaluation_dataset_item.extra,
+        evaluation_dataset_id=evaluation_dataset_item.evaluation_dataset_id,
+    )
+
+    session.add(evaluation_task_item)
+    session.commit()
+
+    return evaluation_task_item
+
+
+@router.delete("/admin/evaluation/dataset-item/{evaluation_dataset_item_id}")
+def delete_evaluation_dataset_item(
+        evaluation_dataset_item_id: int,
+        session: SessionDep,
+        user: CurrentSuperuserDep
+) -> bool:
+    evaluation_dataset_item = session.get(EvaluationDatasetItem, evaluation_dataset_item_id)
+
+    if not evaluation_dataset_item:
+        raise HTTPException(
+            status_code=status.HTTP_404_NOT_FOUND,
+            detail="Evaluation dataset item not found",
+        )
+
+    session.delete(evaluation_dataset_item)
+    session.commit()
+
+    return True
+
+
+@router.put("/admin/evaluation/dataset-item/{evaluation_dataset_item_id}")
+def update_evaluation_dataset_item(
+    evaluation_dataset_item_id: int,
+    updated_evaluation_dataset_item: ModifyEvaluationDatasetItem,
+    session: SessionDep,
+    user: CurrentSuperuserDep
+) -> EvaluationDatasetItem:
+    evaluation_dataset_item = session.get(EvaluationDatasetItem, evaluation_dataset_item_id)
+
+    if not evaluation_dataset_item:
+        raise HTTPException(
+            status_code=status.HTTP_404_NOT_FOUND,
+            detail="Evaluation dataset item not found",
+        )
+
+    evaluation_dataset_item.query = updated_evaluation_dataset_item.query
+    evaluation_dataset_item.reference = updated_evaluation_dataset_item.reference
+    evaluation_dataset_item.retrieved_contexts = updated_evaluation_dataset_item.retrieved_contexts
+    evaluation_dataset_item.extra = updated_evaluation_dataset_item.extra
+    evaluation_dataset_item.evaluation_dataset_id = updated_evaluation_dataset_item.evaluation_dataset_id
+
+    session.commit()
+
+    return evaluation_dataset_item
+
+
+@router.get("/admin/evaluation/dataset/{evaluation_dataset_id}/dataset-item")
```

Comment:
```suggestion
@router.get("/admin/evaluation/datasets/{evaluation_dataset_id}/items")
```

Please follow existing API specifications and use plural noun in the path.

`dataset_items` simpify to `items`.

Reply:
Sure, but only for listing APIs.

Reply:
And for `items`'s simplification, I want to adhere to the same name as the other APIs, rather than a short name.